fenix/app/src/main/java/org/mozilla/fenix/shopping/ReviewQualityCheckFragment.kt

```diff
@@ -51,7 +51,9 @@ class ReviewQualityCheckFragment : BottomSheetDialogFragment() {
                 val bottomSheet =
                     findViewById<View?>(com.google.android.material.R.id.design_bottom_sheet)
                 bottomSheet?.setBackgroundResource(android.R.color.transparent)
-                behavior = BottomSheetBehavior.from(bottomSheet)
+                behavior = BottomSheetBehavior.from(bottomSheet).apply {
+                    peekHeight = resources.displayMetrics.heightPixels / 2
```

Comment:
Could we move this to `onConfigurationChanged` ? I noticed when switching between landscape and portrait the bottom sheet keeps the height from the initial orientation it was created it. If we open the bottom sheet in landscape then switch to portrait without closing, we'd have a very small bottom sheet.

Also, this seems to also solve https://bugzilla.mozilla.org/show_bug.cgi?id=1856378. Nice!

Reply:
<video src="https://github.com/mozilla-mobile/firefox-android/assets/35462038/5cd9fd6a-d95a-4e6e-b758-3ea41c3d0a15"/>

Reply:
Good point, since onConfigurationChange won't be called when bottom sheet is first created, it makes sense to have it in both places as we tested out offline. Updated!